tee-worker/litentry/core/stf-task/sender/src/lib.rs

```diff
@@ -98,6 +98,7 @@ pub struct AssertionBuildRequest {
 	pub who: AccountId,
 	pub assertion: Assertion,
 	pub vec_identity: BoundedVec<Identity, MaxIdentityLength>,
+	pub credential: Credential,
```

Comment:
Question: why do we need to pass `Credential` around? Can't we generate it on the stf-task handler side? (do we have enough information?)